src/goInstallTools.ts

```diff
@@ -148,7 +148,14 @@ export function installTools(missing: Tool[], goVersion: GoVersion): Promise<voi
 		}
 
 		return res.then(sofar => new Promise<string[]>((resolve, reject) => {
+			const opts = {
+				env: envForTools,
+				cwd: toolsTmpDir,
+			};
 			const callback = (err: Error, stdout: string, stderr: string) => {
+				// Make sure to run `go mod tidy` between tool installations.
+				// This avoids us having to create a fresh go.mod file for each tool.
+				cp.execFileSync(goRuntimePath, ['mod', 'tidy'], opts);
```

Comment:
Shouldn't we do this only if `modulesOff` is false?

Reply:
Done.